packages/contracts-bedrock/.env.example

```diff
@@ -0,0 +1,5 @@
+GITHUB_TOKEN=
+SOURCE_FILE="/absolute/path/to/unclaimed_op_drop.txt"
+SANCTIONED_URL="https://raw.githubusercontent.com/ethereum-optimism/sanctioned-addresses/main/list.json"
+SANCTIONED_FILE="/absolute/path/to/sanctioned-addresses.json"
+SAVE_DESTINATION="/absolute/path/to/non-sanctioned-addresses.txt"
```

Comment:
maybe we should prefix all of these with `ONION_` since they are specific to the onion cli?